pkg/deploy/registry/deployconfig/etcd/etcd_test.go

```diff
@@ -1,135 +0,0 @@
-package etcd
```

Comment:
> @deads2k for clarification, we don't need this test anymore or it moves elsewhere or we already test this in different place?

We don't need it because its just re-running the generic etcd storage testing.